ui/components/MesheryPerformance/index.js

```diff
@@ -467,12 +482,10 @@ const MesheryPerformanceComponent = (props) => {
 
   const handleProfileUpload = (body, generateNotif, cb) => {
     if (generateNotif) props.updateProgress({ showProgress: true });
-
-    dataFetch(
-      '/api/user/performance/profiles',
-      { method: 'POST', credentials: 'include', body: JSON.stringify(body) },
-      (result) => {
-        if (typeof result !== 'undefined') {
+    savePerformanceProfile({ body })
```

Comment:
check here https://github.com/meshery/meshery/blob/master/ui/rtk-query/performance-profile.js#L30. pass the body argument